### DISCUSSION TITLE
Masking in MALT

### DISCUSSION THREAD

**Discussion by benny7770** [bnpr/Malt#423] [Q&A] [answered]
Hi ya ! pretty straight forward question but have not found an answer in documentation. 

I have two collections, I want one collection to mask another using the holdout function. is this possible and if not what is the process to "holdout" in MALT.  

Cheers

**Comment by pragma37** [comment, marked as answer]
Malt doesn't extract collection information from Blender yet.
You can get a "holdout" material by setting the *Color* alpha to 0, without enabling the *Transparency* option in the Material:
![imagen](https://user-images.githubusercontent.com/36610999/196544329-96ca3af7-fc29-41e8-a16a-290bc04fe301.png)

**Comment by benny7770** [comment]
great thanks very much
